xdem/spatial_tools.py

```diff
@@ -66,7 +66,7 @@ def subtract_rasters(first_raster: Union[str, gu.georaster.Raster], second_raste
             )
 
     # Reproject the non-reference and subtract the two rasters.
-    difference = first_raster.data - second_raster.reproject(second_raster, resampling=resampling_method).data if \
+    difference = first_raster.data - second_raster.reproject(first_raster, resampling=resampling_method).data if \
```

Comment:
since this is fixing a bug, ideally this should be a different PR

Reply:
This is a merge with a small correction that I did earlier.

Reply:
;)